public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Print Link3d</title>
```

Comment:
Print Link3d or PrintLink3d?

Reply:
Hmm I suppose we've been going with PrintLink3d. I'll change it